Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+pipeline {
+
+    agent { label "codejail-worker" }
+
+    options {
+        timeout(30)
+    }
+    stages {
+        stage('Install tox') {
+            steps {
+                sh '''
+                virtualenv tox_env;
+                source tox_env/bin/activate;
+                pip install -r requirements/tox.txt
+                '''
+            }
+        }
+
+        stage('Run the tests for each sandbox') {
+            parallel{
+                stage('Run tests with python 2.7') {
+                    steps {
+                        script {
+                            try {
+                                sh '''
+                                source tox_env/bin/activate;
+                                export CODEJAIL_TEST_USER=sandbox;
+                                export CODEJAIL_TEST_VENV='/home/sandbox/codejail_sandbox-python-2-7';
```

Comment:
Maybe set these in an `environment` block for each stage instead?

Reply:
Done